.github/workflows/ci.yml

```diff
@@ -32,4 +32,12 @@ jobs:
           node-version: ${{ matrix.node-version }}
           cache: pnpm
       - run: pnpm install --frozen-lockfile
-      - run: pnpm test
+      - name: lint
+        if: ${{ !cancelled() }}
```

Comment:
do we really need `if: ${{ !cancelled() }}` everywhere?

Reply:
this allows a single run to output all failed commands

Reply:
```suggestion
        if: ${{ !cancelled() }} # allows checks to run if one fails so we can put them all in the same job
```